docs/reference/asyncapi-rules.md

```diff
@@ -0,0 +1 @@
+# AsyncAPI Rules
```

Comment:
Is this file incomplete?

Reply:
@fmvilas Before diving into the documentation part of the PR, I was expecting some comments from @philsturgeon with regards to the ruleset first.

This file will eventually list all the available rules.

Reply:
@fmvilas @philsturgeon The file now list all the proposed rules. Could you please take a look at them?

Every re-wording proposal would be greatly appreciated! ;-)